soil/util.py

```diff
@@ -1,41 +1,26 @@
-from django.conf import settings
-from django.utils import importlib
-from soil import FileDownload, DownloadBase
+from soil import DownloadBase, CachedDownload, FileDownload
 from soil.exceptions import TaskFailedError
 from soil.heartbeat import heartbeat_enabled, is_alive
 
 
-def get_default_backend():
+def expose_cached_download(payload, expiry, **kwargs):
     """
-    Get default export class. To override set it in your settings.
-    
-    This defaults to FileDownload.
+    Expose a cache download object.
     """
-    if hasattr(settings, "SOIL_BACKEND"):
-        # Trying to import the given backend, in case it's a dotted path
-        backend = settings.SOIL_BACKEND
```

Comment:
After this change you can delete all references to `SOIL_BACKEND`. Maybe a separate PR